src/02_datatypes.py

```diff
@@ -14,8 +14,10 @@
 # Write a print statement that combines x + y into the integer value 12
 
 # YOUR CODE HERE
-
+x + int(y)
```

Comment:
In order for x + y to add up to 12, you first need to store the Values in x and y. 
x = 5
y = "7" 
Wrapping quotes around 7 makes it a **String** instead of an Integer. 
Then you would print it using: print(x + int(y))
using int converts **y** into an **int** for that line. It doesn't change the DataType overall unless you store it as an **int**